xbmc/Application.cpp

```diff
@@ -3568,6 +3572,8 @@ void CApplication::OnPlayBackStopped()
   CDarwinUtils::EnableOSScreenSaver(true);
 #endif
 
+  CServiceBroker::GetPVRManager().OnPlaybackStopped(m_itemCurrentFile);
+
```

Comment:
wouldn't it be cleaner to register PVRmanger and all the other components above to the respective announcement from CAnnouncementManager below? (same with onStart, etc). Would get rid of all the ifdeffery. Just a general question for some future refactoring that came to my mind when looking at this PRs changes.